File: Interoperability.java
package com.MyJava;

import com.MyKotlen.MyCustomFileName;

public class Interoperability
{
    public static void main(String[] args)
    {
      MyCustomFileName.sum(32,43);

      //new kotlen file name

        //System.out.println( InterOperabilityKt.add(32,323));
 //       System.out.println("From the java Class the com.MyKotlen.sum is :"+com.MyKotlen.sum);
    }
    public static int  sum2(int length,int bredth)
    {
        int a,b;
        a=length;
        b=bredth;
        return a*b;
    }
}

File: InterOperability.kt
@file:JvmName("MyCustomFileName")

package com.MyKotlen

import com.MyJava.Interoperability

fun main(args:Array<String>)
{
    var g= Interoperability.sum2(32,32)
println("The area of the rectangle is $g")
}
fun sum(a:Int,b:Int):Unit
{
println ("The com.MyKotlen.sum of these two numbers $a and $b is ${a+b}")
}

/**
 * we can also use another saytax like
 * @file.JvmName("MyFirstCoustemFileName")
 * then we can call the kotlen fxns from the java class using this name MyCustomFileName
 * like int java class -> MyCustomFileName.com.MyKotlen.sum;
 *
 */
